libcnb-data/src/generic.rs

```diff
@@ -0,0 +1,2 @@
+/// Generic TOML metadata.
+pub type GenericMetadata = Option<toml::value::Table>;
```

Comment:
Could `GenericMetadata` also be used here? (ie made the default and the tests simplified)
https://github.com/heroku/libcnb.rs/blob/7e800211d87a42f6ad60cfb626ca570f47539e7c/libcnb-data/src/layer_content_metadata.rs#L8
https://github.com/heroku/libcnb.rs/blob/7e800211d87a42f6ad60cfb626ca570f47539e7c/libcnb-data/src/layer_content_metadata.rs#L43

Reply:
Oh also here:
https://github.com/heroku/libcnb.rs/blob/785602d2a09dcf03c6c3be25354591ecbdec8639/libcnb-package/src/lib.rs#L39

Reply:
Ah, that's cool! Changed it to use `GenericMetadata` (implicitly) as well. Thanks! 👍🏻